docs/kms_secretlock.md

```diff
@@ -0,0 +1,143 @@
+# Aries New Secure KMS, SecretLock and Crypto capabilities
+
+The aries-framework-go project has a mandate to protect keys from being compromised during usage or storage of keys. 
+This goal can achieved by hiding raw key bytes from being exchanged between the user and the framework. Also, keys
+should be secured and locked by a master key that is knwon to the user and protected by a passphrase. In order 
```

Comment:
Paraphrase is only an example protection.

Reply:
updated text to mention it as an example